ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/HD/LMDB.hs

```diff
@@ -132,8 +145,11 @@ defaultLMDBLimits = LMDB.defaultLimits
   , LMDB.maxReaders = 16
   }
 
-newtype DbState  = DbState
-  { dbsSeq :: WithOrigin SlotNo  -- TODO a version field
+-- TODO a version field.
+data DbSettings = DbSettings
```

Comment:
The settings are currently stored in the database itself, but this may not be sensible: We can only access them *after* we have opened both the `LMDB` environment and corresponding `LMDB` settings database, while these settings could be required *before* opening the environment.

Reply:
Good point. Shouldn't the settings be passed in a configuration file? 

Also do these settings refer exclusively to the LMDB settings? If so we might want to rename this data type.

Reply:
We decided to keep the settings around, such that we can use it in the future.